virtcontainers/virtiofsd_test.go

```diff
@@ -64,6 +65,7 @@ func TestVirtiofsdStart(t *testing.T) {
 				},
 			}
 			var ctx context.Context
+			err = os.MkdirAll(filepath.Dir(v.socketPath), testDirMode)
```

Comment:
As shown on the PR, passing invalid args would be a good test that this is working. Could you find a way to create a unit test for that to guarantee debugging always works (distressing if it doesn't - when you need it, you need it ;)

Reply:
Actually, related with this specific case, I needed to create the directory before if not the start function will fail because the runtime will fail open the socket for virtiofsd, in the use case the directory already exists, I wonder but I wonder if the directory does not exist the function should create before open the socket, what do you think?

Reply:
Sorry - I'm not commenting on this `mkdir` specifically, I'm just saying that if it's possible, it would be good to add a test to ensure debug output is actually captured from clh/virtiofs. Using some invalid params is one way, but there may be better ones ;)